src/sections/stats/components/RecentTradesTable/RecentTradesTable.utils.tsx

```diff
@@ -31,154 +41,165 @@ export const useRecentTradesTable = (data: TRecentTradesTableData) => {
   const columnVisibility: VisibilityState = {
     isBuy: true,
     account: isDesktop,
-    totalValue: true,
+    amountIn: true,
     trade: isDesktop,
     date: isDesktop,
   }
 
-  const columns = useMemo(
-    () => [
-      accessor("isBuy", {
-        id: "type",
-        header: t("stats.overview.table.trades.header.action"),
-        sortingFn: (a, b) => (a.original.isBuy === b.original.isBuy ? 1 : -1),
-        cell: ({ row }) => (
-          <div
-            sx={{
-              flex: "row",
-              gap: 8,
-              align: "center",
-              justify: ["start", "center"],
-            }}
-          >
-            <Icon
-              sx={{ color: "darkBlue300" }}
-              icon={row.original.isBuy ? <BuyIcon /> : <SellIcon />}
-            />
-            {isDesktop ? (
-              <Text color="white">{row.original.isBuy ? "Buy" : "Sell"}</Text>
-            ) : (
-              <div sx={{ flex: "row", align: "center", gap: 6 }}>
-                <Icon
-                  size={18}
-                  icon={<AssetLogo id={row.original.assetInId} />}
-                />
-                <Text>{row.original.assetInSymbol}</Text>
-
-                <Icon
-                  sx={{ color: "brightBlue600" }}
-                  css={{
-                    transform: row.original.isBuy
-                      ? "rotate(180deg)"
-                      : undefined,
-                  }}
-                  icon={<TradeIcon />}
-                />
-
-                <Icon
-                  size={18}
-                  icon={<AssetLogo id={row.original.assetOutId} />}
-                />
-                <Text>{row.original.assetOutSymbol}</Text>
-              </div>
-            )}
-          </div>
-        ),
-      }),
-      accessor("account", {
-        id: "account",
-        header: t("stats.overview.table.trades.header.account"),
-        sortingFn: (a, b) =>
-          a.original.account.localeCompare(b.original.account),
-        cell: ({ row }) => (
-          <Text tAlign={isDesktop ? "center" : "right"} color="white">
-            {shortenAccountAddress(row.original.account, 10)}
-          </Text>
-        ),
-      }),
-      accessor("totalValue", {
-        id: "totalValue",
-        header: t("stats.overview.table.trades.header.totalValue"),
-        sortingFn: (a, b) =>
-          a.original.totalValue.gt(b.original.totalValue) ? 1 : -1,
-        cell: ({ row }) =>
-          isDesktop ? (
-            <Text tAlign="center" color="white">
-              <DisplayValue value={row.original.totalValue} isUSD />
-            </Text>
+  const spot = useSpotPrices(
```

Comment:
Could you do those calculations in the file where you prepare the data for the table, including the spot price and hydra account address?